### PR TITLE
Change Skylake Spectre_V2 default to Retpolines (#1659626)

### PR DESCRIPTION
**Pull request by poncovka**
Set the boot option `spectre_v2` to `retpoline` on Intel Skylake systems.

Resolves: rhbz#1659626

### CODE REVIEW

File: pyanaconda/bootloader.py [hunk 1]
```diff
@@ -826,6 +826,12 @@ def set_boot_args(self, *args, **kwargs):
         if flags.cmdline.get("fips") == "1" and boot_device:
             self.boot_args.add("boot=%s" % self.stage2_device.fstabSpec)
 
+        #
+        # Spectre
```
Comment (jkonecny12) [inline pyanaconda/bootloader.py]: I would rename this to something like ``Mitigation to Spectre/Meltdown``

File: pyanaconda/isys/__init__.py [hunk 2]
```diff
@@ -99,6 +99,27 @@ def isLpaeAvailable():
...
+def get_cpu_model():
+    """Get the cpu model number."""
+    with open("/proc/cpuinfo", "r") as f:
+        for line in f:
+            name, value = line.split(":")
+
+            if name.strip() == "model":
+                return int(value.strip())
+
+    return None
```
Comment (jkonecny12) [inline pyanaconda/isys/__init__.py]: I hoped that python will have something in the standard library but I wasn't able to find anything :(.

Please add information about output format to the function documentation.

File: pyanaconda/bootloader.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/bootloader.py]: Fixed.

File: pyanaconda/isys/__init__.py (hunk 2 above)
Comment (poncovka) [reply pyanaconda/isys/__init__.py]: Fixed.

Comment (jkonecny12) [reply pyanaconda/isys/__init__.py]: I think there should be also `an integer value *or None*`. The None part possibility is missing here.

Comment (poncovka) [reply pyanaconda/isys/__init__.py]: It should be fixed now. Thanks!

**Comment by poncovka** [comment]
Tested with my cpu model listed in the set of Skylake models.

**Pull request merged**